_posts/ja/newsletters/2022-12-07-newsletter.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Bitcoin Optech Newsletter #229'
+permalink: /ja/newsletters/2022/12/07/
+name: 2022-12-07-newsletter-ja
+slug: 2022-12-07-newsletter-ja
+type: newsletter
+layout: newsletter
+lang: ja
+---
+今週のニュースレターは、エフェメラル・アンカーの実装に加えて、Bitcoin Core PR Review Clubミーティングの概要や、
+新しいリリースおよびリリース候補の発表、人気のあるBitcoinインフラストラクチャプロジェクトの注目すべき変更など
+恒例のセクションも含まれています。
+
+## ニュース
+
+- **<!--ephemeral-anchors-implementation-->エフェメラル・アンカーの実装:** Greg Sandersは、
+  エフェメラル・アンカー（[ニュースレター #223][news223 anchors]参照）に関する彼のアイディアを実装したことを
+  Bitcoin-Devメーリングリストに[投稿しました][sanders ephemeral]。
+  [アンカー・アウトプット][topic anchor outputs]は、Bitcoin Coreの[CPFP carve outs][topic cpfp carve out]によって利用可能になった
+  既存の技術で、LNのコントラクトに参加する両参加者が[CPFP][topic cpfp]によりそのコントラクトに関連する
+  トランザクションの手数料を引き上げられることを保証するためにLNプロトコルで使用されています。
+  アンカー・アウトプットにはいくつか欠点があります。根本的なものもあれば（[ニュースレター #224][news224 anchors]参照）、
+  対処可能なものもあります。
+
+    エフェメラル・アンカーは、[v3トランザクションリレーの提案][topic v3 transaction relay]に基づいており、
+    v3トランザクションは`OP_TRUE`スクリプトへのゼロ値の支払いのアウトプットを含めることができます。
+    これにより、ネットワーク上の誰でも使用可能なUTXOを持っていれば、CPFPでそのトランザクションの手数料を引き上げることができます。
+    手数料を引き上げる子トランザクションは、それ自体が他の使用可能なUTXOによってRBFによる手数料の引き上げが可能です。
+    v3トランザクションの他のリレーポリシーと組み合わせることで、時間的な制約のあるコントラクトプロトコルのトランザクションに対する
+    [トランザクションPinning攻撃][topic transaction pinning]に関してポリシーベースの懸念が解消されることが期待されます。
+
+    さらに、誰でもエフェメラル・アウトプットを含むトランザクションの手数料を引き上げられるため、
+    2人以上が参加するコントラクトプロトコルで利用可能です。
+    既存のBitcoin Coreのcarve outルールは、参加者が2人の場合のみ機能し、
+    これを増やすための[これまでの試み][bitcoin core #18725]では参加者の任意の上限を必要としました。
+
+    Sandersのエフェメラル・アンカーの[実装][bitcoin core #26403]により、
+    提案者によって以前実装された他のv3トランザクションリレー動作と共に、このアイディアのテストができるようになりました。
+
+## Bitcoin Core PR Review Club
+
+*この毎月のセクションでは、最近の[Bitcoin Core PR Review Club][]ミーティングを要約し、
+重要な質問と回答のいくつかに焦点を当てます。
+以下の質問をクリックしてミーティングでの回答の要約を確認してください。*
+
+[Bump unconfirmed ancestor transactions to target feerate][review club 26152]は、
+Xekyo (Murch)とglozowによるPRで、未承認のUTXOをインプットとして選択した場合のウォレットの手数料計算の精度を向上させるものです。
+このPRがない場合、インプットとして使用されるいくつかの未承認トランザクションの手数料率が、
+構築されるトランザクションの手数料率より低い場合、低すぎる手数料が設定されてしまいます。
+このPRでは、このような低手数料のソーストランザクションを
+新しいトランザクションのターゲットと同じ手数料率に「促進させる」ために十分な手数料を追加することでこの問題を解決します。
+
+このPRがなくても、コイン選択プロセスは、低手数料率の未承認トランザクションの使用を避けようとする点にご注意ください。
+このPRは、これを避けることができない場合に有用です。
```

Comment:
I think it would be better to add some words to clarify the meaning of the second `this`.
```suggestion
このPRは、これ(低すぎる手数料が設定されてしまう問題)を避けることができない場合に有用です。
```

Reply:
@AkioNak thanks!